### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_Select Source-Target_Schema Compare_AI4D: Name property is missing for all controls of Schema compare screen. 

### ISSUE THREAD

**Issue by ShwetaBaglane** [microsoft/azuredatastudio#6850]
[“Check out Accessibility Insights! !](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #A11y_ADS_Schema_Dacpac_Backup; #Azure Data Studio; #DesktopApp; #Win10; #MAS4.1.2;

**Environment Details:**
Application Application Name: Azure Data Studio
Application Version: 1.10.0-insider (user setup)
Commit: 1ff9a2ec4c7b88be01666bdd470bbbb4ab95716e
Date: 2019-08-09T07:49:00.254Z
VS Code 1.37.0
Electron: 4.2.7
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
MAS References: MAS4.1.2

**Repro Steps:**
Launch Azure Data Studio application.
Connect to server with windows credentials.
Navigate to Extensions from left menu.
Search & install Schema Compare extension.
Navigate to Connections from left menu.
Expand the server connected in step 2.
Right click an on prem database and select Schema compare from Context menu.
Navigate till Select Source or Select Target & hit enter.
Launch accessibility Insights tool and apply it on schema compare screen.

**Actual:**
Name property is missing for all controls of Schema compare screen.

**Expected:**
Name property should be present for all controls as per label present against that control.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: (https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader users will not get to know the context/ label for each control.

**MAS Reference:**
MAS4.1.2
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

**Attachment for Reference:**
![image](https://user-images.githubusercontent.com/53556874/63430044-44ebe100-c439-11e9-97da-84eaa6d5ad6d.png)

**Comment by udeeshagautam** [comment]
I have reopened this issue so the Accessibility team can add comments.

**Comment by fsteffi** [comment]
Checked and verified in below version the bug is no longer reproduced and hence closing the bug.

Version Details:
Version: 1.12.0-insider
Commit: 8e1a2248e47925eb825b0ce037b96b6c87798a8a
Date: 2019-10-04T07:52:03.973Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

![#6850A11y_AzureDataStudio(Schema,Dacpac,Backup)_Select Source-Target_Schema Compare_AI4D](https://user-images.githubusercontent.com/24580570/66801396-92b33000-ef36-11e9-8ac0-d8eccffa6367.png)

**Comment by udeeshagautam** [comment]
Closing after Accessibility team's confirmation.
